sdk/identity/azure-identity/azure/identity/_internal/decorators.py

```diff
@@ -26,8 +26,11 @@ def wrapper(*args, **kwargs):
             )
             if _LOGGER.isEnabledFor(logging.DEBUG):
                 try:
-                    base64_meta_data = token.token.split(".")[1].encode("utf-8") + b"=="
-                    json_bytes = base64.decodebytes(base64_meta_data)
+                    base64_meta_data = token.token.split(".")[1]
+                    padding_needed = 4 - (len(base64_meta_data) % 4)
```

Comment:
A more concise way to achieve the same result while avoiding the potential for the padding_needed value to be 4.

```suggestion
                    padding_needed = -len(base64_meta_data) % 4
```